### PR TITLE
fix: multi-entry bundle with environment variables

### PR DESCRIPTION
**Pull request by jbedard**

### CODE REVIEW

File: webpack/private/webpack_bundle.bzl [hunk 1]
```diff
@@ -221,7 +221,10 @@ def _impl(ctx):
     if ctx.attr.chdir:
         env["JS_BINARY__CHDIR"] = ctx.attr.chdir
     for (key, value) in ctx.attr.env.items():
-        env[key] = " ".join([expand_variables(ctx, exp, attribute_name = "env") for exp in expand_locations(ctx, value, [ctx.attr.entry_point] + ctx.attr.srcs + ctx.attr.deps + ctx.attr.data).split(" ")])
+        env[key] = " ".join([
+            expand_variables(ctx, exp, attribute_name = "env")
+            for exp in expand_locations(ctx, value, [ctx.attr.entry_point] if ctx.attr.entry_point else ctx.attr.entry_points.keys() + ctx.attr.srcs + ctx.attr.deps + ctx.attr.data).split(" ")
```
Comment (gregmagolan) [inline webpack/private/webpack_bundle.bzl:226]: add some brackets so the order of operations is clear?
```
([ctx.attr.entry_point] if ctx.attr.entry_point else ctx.attr.entry_points.keys()) + ctx.attr.srcs + ctx.attr.deps + ctx.attr.data
```

Comment (gregmagolan) [reply webpack/private/webpack_bundle.bzl:226]: actually; probably better to assign to a variable at the top of the for look to make it readable

Comment (jbedard) [reply webpack/private/webpack_bundle.bzl]: Updated

**Review by gregmagolan — APPROVED** [review]
🏄‍♀️

**Pull request merged**